app/models/rdv_plan.rb

```diff
@@ -14,6 +14,7 @@ class RdvPlan < ApplicationRecord
   delegate :organisation, to: :motif
 
   validate :return_url_is_authorized
+  validates :user_id, :planning_agent_id, presence: true
```

Comment:
Discuté ensemble : un ajout de `NOT NULL` en base serait une garantie plus solide.

Reply:
Mais pourquoi pas garder cette validation, même si elle génère un message d'erreur potentiellement peu utile : "user_id is missing" -> alors que l'erreur c'est que le user passé est invalide.